src/test/java/uk/gov/ida/eventemitter/utils/TestEvent.java

```diff
@@ -0,0 +1,79 @@
+package uk.gov.ida.eventemitter.utils;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import junit.framework.Test;
+import org.joda.time.DateTime;
+import uk.gov.ida.eventemitter.AuditEvent;
+import uk.gov.ida.eventemitter.Event;
+import uk.gov.ida.eventemitter.EventDetailsKey;
+
+import java.util.EnumMap;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.UUID;
+
+public final class TestEvent implements AuditEvent {
+
+    @JsonProperty
+    private String eventId;
+
+    @JsonProperty
+    private DateTime timestamp;
+
+    private TestEvent() {}
+
+    public TestEvent(final String eventId,
+                     final DateTime timestamp) {
+        this.eventId = eventId;
+        this.timestamp = timestamp;
+    }
+
+    @Override
+    public String getEventId() {
+        return eventId;
+    }
+
+    @Override
+    public DateTime getTimestamp() {
+        return timestamp;
+    }
+
+    public void setEventId(String eventId) {
+        this.eventId = eventId;
+    }
+
+    public void setTimestamp(DateTime timestamp) {
+        this.timestamp = timestamp;
+    }
+
+    @Override
+    public String getJSONString () {
+        final StringBuffer sb = new StringBuffer("TestAuditEvent{");
+        sb.append("eventId=").append(eventId);
+        sb.append(", timestamp=").append(timestamp);
+        sb.append('}');
+        return sb.toString();
```

Comment:
This doesn't look like JSON to me. Is this intentional?

Reply:
Have now delegated json serialisation to an ObjectMapper